### PR TITLE
Annotator template for Wikidata

### PR DESCRIPTION
**Pull request by egonw**

### CODE REVIEW

File: src/pyBiodatafuse/annotators/wikidata.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+import pandas as pd
+from SPARQLWrapper import JSON, SPARQLWrapper
+
+from pyBiodatafuse.utils import collapse_data_sources, get_identifier_of_interest
+
+
+def get_version_wikidata() -> dict:
+
+    wikidata_version = {"wikidata_version": "0"}
```
Comment (tabbassidaloii) [inline src/pyBiodatafuse/annotators/wikidata.py:16]: I think wikidata doesn't provide version, but is it a common practice to use 0?

File: src/pyBiodatafuse/annotators/wikidata.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+import os
+from string import Template
+
+import pandas as pd
+from SPARQLWrapper import JSON, SPARQLWrapper
+
+from pyBiodatafuse.utils import collapse_data_sources, get_identifier_of_interest
+
+
+def get_version_wikidata() -> dict:
```
Comment (tabbassidaloii) [inline src/pyBiodatafuse/annotators/wikidata.py:14]: please provide a docstring

File: src/pyBiodatafuse/annotators/wikidata.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+    wikidata_version = {"wikidata_version": "0"}
+
+    return wikidata_version
+
+
+def get_gene_literature(bridgedb_df: pd.DataFrame):
```
Comment (tabbassidaloii) [inline src/pyBiodatafuse/annotators/wikidata.py:21]: please provide a docstring

File: src/pyBiodatafuse/annotators/wikidata.py [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+            "size": len(gene_list),
+            "time": time_elapsed,
+            "date": current_date,
+            "url": "https://sparql.wikidata.org/sparql",
+        },
+    }
+
+    return data_df, wikidata_metadata
+
+def foo():
```
Comment (tabbassidaloii) [inline src/pyBiodatafuse/annotators/wikidata.py:96]: to be updated?

File: src/pyBiodatafuse/annotators/wikidata.py [hunk 5]
```diff
@@ -63,15 +63,18 @@ def get_gene_literature(bridgedb_df: pd.DataFrame):
 
     # Organize the annotation results as an array of dictionaries
     intermediate_df = pd.concat(results_df_list)
+    intermediate_df = intermediate_df.rename(columns={"article": "wikidata_id"})
```
Comment (tabbassidaloii) [inline src/pyBiodatafuse/annotators/wikidata.py:66]: @egonw please see the scripts added here (L66-L77)

File: src/pyBiodatafuse/annotators/wikidata.py (hunk 1 above)
Comment (egonw) [reply src/pyBiodatafuse/annotators/wikidata.py:16]: No, was just a template. I am not sure Wikidata has a verion. it changes all the time. I'll come back on this.

Comment (egonw) [reply src/pyBiodatafuse/annotators/wikidata.py:15]: I'm using `str(datetime.datetime.now())[0:16]` now.

File: src/pyBiodatafuse/annotators/wikidata.py (hunk 4 above)
Comment (egonw) [reply src/pyBiodatafuse/annotators/wikidata.py]: To be removed :)

**Review by tabbassidaloii — APPROVED** [review]

**Pull request merged**